examples/gke_cluster/variables.tf

```diff
@@ -0,0 +1,19 @@
+variable "castai_api_token" {
+  default = "<<replace-this-with-your-token>>"
+}
+
+variable "castai_url" {
+  default = "<<replace-this-with-castai-api-url>>"
```

Comment:
set default to PROD API endpoint - this will not be subject to change most of the time, or just don't implement as variable at all, leaving default value.

Reply:
good point, removed at all, using prod by default